13-loops/exercises/08-lucky-number-exercises/03-double-guesses/main.go

```diff
@@ -8,19 +8,82 @@
 
 package main
 
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
 // ---------------------------------------------------------
 // EXERCISE: Double Guesses
 //
-//  Let the player guess two numbers instead of one.
+//  Let the player guess1 two numbers instead of one.
 //
 // HINT:
 //  Generate random numbers using the greatest number
-//  among the guessed numbers.
+//  among the guess1ed numbers.
 //
 // EXAMPLES
 //  go run main.go 5 6
 //  Player wins if the random number is either 5 or 6.
 // ---------------------------------------------------------
 
+const (
+	maxTurns = 5 // less is more difficult
+	usage    = `Welcome to the Lucky Number Game! 🍀
+
+The program will pick %d random numbers.
+Your mission is to guess1 one of those numbers.
+
+The greater your number is, harder it gets.
+
+Wanna play?
+`
+)
+
 func main() {
+	rand.Seed(time.Now().UnixNano())
+
+	args := os.Args[1:]
+
+	if len(args) != 2 {
+		fmt.Printf(usage, maxTurns)
+		return
+	}
+
+	guess1, err1 := strconv.Atoi(args[0])
```

Comment:
use small names like g instead-of guess1 
also, here use err as name and then reuse err again rather than declaring new err2

Reply:
Fixed